app/ui/browser-modern/model/page-ui-state.js

```diff
@@ -16,6 +16,7 @@ const PageUIState = Immutable.Record({
   zoomLevel: 0,
   searchVisible: false,
   pinned: false,
+  webContentsId: undefined,
```

Comment:
Not sure this makes sense to be in uiState. Why not keep it directly on the page model?